app/main/views/questionnaire.py

```diff
@@ -119,13 +125,31 @@ def submit_answers(eq_id, form_type, collection_id):
         return redirect_to_questionnaire_page(eq_id, form_type, collection_id, invalid_location)
 
 
+@action_blueprint.route('<location>/<question>/add', methods=["POST"])
+@login_required
+def add_answer(eq_id, form_type, collection_id, location, question):
+    answer_store = get_answer_store(current_user)
```

Comment:
The reason for passing it in is that it is easier to use a mock answer store during unit tests. If you can show me how to mock the global call to get_answer_store within questionnaire_manager.add_answer, then I'm happy to move it.